sirepo/job_supervisor.py

```diff
@@ -303,6 +363,10 @@ def __db_write(self):
             dataFileKey=req.content.pop('dataFileKey')
         )
 
+    @classmethod
+    async def _receive_api_ownJobs(cls, req):
+        return cls._get_running_jobs()
```

Comment:
How is this restricted just to the users's jobs?

Please add a test to validate this question.